Assets/MRTK/Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -1305,14 +1305,12 @@ public void RaisePointerUp(IMixedRealityPointer pointer, MixedRealityInputAction
                 var inputData = ExecuteEvents.ValidateEventData<InputEventData>(eventData);
                 Debug.Assert(inputData.MixedRealityInputAction != MixedRealityInputAction.None);
 
-                var inputHandler = handler as IMixedRealityInputHandler;
-                if (inputHandler != null)
+                if (handler is IMixedRealityInputHandler inputHandler && !inputHandler.IsNull())
```

Comment:
Again a bit confused why we have IsNull() checks here but not other places



A bit confused about the !boundsProvider.IsNull() check since you didn't do this in other places.

ie in GameObjectExtension:

-var requireComponentAttribute = attribute as RequireComponent;
-if (requireComponentAttribute != null)
         +if (attribute is RequireComponent requireComponentAttribute)

Reply:
Same answer as https://github.com/microsoft/MixedRealityToolkit-Unity/pull/9129#discussion_r556194562

In this case (`IMixedRealityInputHandler`), there are a bunch of `MonoBehaviour`-based implementers

![image](https://user-images.githubusercontent.com/3580640/104392895-f9d3e180-54f7-11eb-85b5-78f56a79d037.png)